### DISCUSSION TITLE
I would like to see the apps path

### DISCUSSION THREAD

**Discussion by jddcef** [NexovaDev/UpdateHub#280] [Ideas]
Hi, could you show the app's path e.g. c:\program files\blah\etc.exe

Because it's hard to know where this app was found, so I can have better understanding of which app it is updating more precisely.

Thanks.

**Comment by mcbyte-it** [comment]
Same here, I see some app names that have an update while I know I have them updated.

It will be useful to have links even to open Explorer to the app install location, and also another button to run the app main executable.

**Comment by onyx-nxt** [comment]
Where would we show this path?

**Reply by jcastilloalonso** [reply]
I think the easiest/most direct would be in the application details so you can show more information about the detected version and path for it.

**Reply by onyx-nxt** [reply]
I'll see if it's possible first. You Want like a "Open in Explorer" button in the more info popup?

**Reply by jddcef** [reply]
This is a tool for advanced users, so don't be shy to show more info than less. Dump as much info as you have in a raw format even, in some random button that opens a text box or just copies it to clipboard if it's really much work,  while decision is being made on which info to wire up to the visible GUI interface.
